recipes/lazy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "lazy" %}
+{%set version = "1.2" %}
+{%set compress_type = "zip" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "127ea610418057b953f0d102bed83f2c367be13b59f8d0ddf3b8a86c7d31b970" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - lazy
+    - lazy.tests
+
+about:
+  home: https://github.com/stefanholek/lazy
+  license: BSD
```

Comment:
😢 

Maybe we should just merge it then. The author seems totally unresponsive. What more can we do?